### DISCUSSION TITLE
Example of using custom fonts

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#215] [Developer Feedback]
Hello.

It's stated that [widget.TEXT](https://docs.zepp.com/docs/reference/device-app-api/newAPI/ui/widget/TEXT/) could use custom fonts.
Also it's mentioned in [folder structure](https://docs.zepp.com/docs/guides/architecture/folder-structure/)

But in [repo](https://github.com/zepp-health/zeppos-samples/tree/main/application/2.0/calories/assets/gts) there are no custom fonts and in code there are no mentions for the custom fonts.

When building package ttf placed in /assets/target is not included in package.

**Comment by MyoungXUE** [comment]
@kartun83 
Hi, thanks for your suggestion, we will add the use case for custom fonts in the official example as soon as possible!

**Comment by MyoungXUE** [comment]
Hi, I have added an example of using custom fonts, please refer to

https://github.com/zepp-health/zeppos-samples/blob/main/application/2.0/showcase/page/ui/widget/text.js
